README.md

```diff
@@ -204,9 +168,7 @@ To run the unit tests, simply run:
 
 ### Running Code Coverage
 
-To generate the code coverage, you first must install
-[node-jscoverage](https://github.com/visionmedia/node-jscoverage). The easist
-way to do this is run:
+To generate the code coverage, you first must install [node-jscoverage](https://github.com/visionmedia/node-jscoverage). The easist way to do this is run:
```

Comment:
is that still a thing? the linked repo is 12 years old and last update was 5 years ago with "don't use it" :smile:

Reply:
maybe we can remove that. I could imagine, that @cb1kenobi doesn't care about

Reply:
Remove this entire code coverage block. End users don't care.